tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -16,9 +16,85 @@
 Documentation  Test 1-02 - Docker Pull
 Resource  ../../resources/Util.robot
 Suite Setup  Conditional Install VIC Appliance To Test Server
-Suite Teardown  Cleanup VIC Appliance On Test Server
+Suite Teardown  Local Teardown
```

Comment:
don't do that, it is not appropriate for any test but the new one.  Just add a [Teardown] at the end of the specific test

Reply:
Didn't know about `Teardown`